src/main/java/mate/controller/GetAllDriversController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/all")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/drivers")
```
don't need to add all for such controllers